### DISCUSSION TITLE
Requesting sbt 0.13.16 deb file

### DISCUSSION THREAD

**Discussion by sujit-baniya** [sbt/sbt#7027] [Q&A]
Can anyone please suggest me the reference to download the sbt v0.13.16 version? https://dl.bintray.com/sbt/debian/sbt-0.13.16.deb is down and now I can't access it anymore

**Comment by eed3si9n** [comment]
They are under https://repo.scala-sbt.org/scalasbt/debian
See also https://www.scala-sbt.org/1.x/docs/Installing-sbt-on-Linux.html
